### DISCUSSION TITLE
In need of a mentor.

### DISCUSSION THREAD

**Discussion by Edwardius** [vercel/next.js#44915] [Help]
Hi, 

I'm using Next.js with Tailwindcss, and I was wondering if anyone could give me a quick mentoring session to look over my website and the code behind it. I also have a question about removing elements from the DOM after they have vanished which I would also like answered (my hamburger menu disappears from sight but not from the DOM).

I'm fairly new to frontend development but not software development in general, so I think one mentoring session is enough to get me going :).

Thanks, here's my [website](https://www.ededdyedward.com/) and my [code](https://github.com/Edwardius/ed-eddy-edward). I haven't had time to make a fav icon, but I'll get that done some time down the road.

**Comment by abdullahtariq1171** [comment]
I just checked your website it removes the the the hamburger from DOM. See attached gif 

![abdullah-01-16-2023 at 15 13 15](https://user-images.githubusercontent.com/5623766/212653605-03ead7ef-30df-4fb0-a387-b80de5608c0b.gif)


Happy to answer your questions.

**Reply by Edwardius** [reply]
Hi Abdullah,

Sorry, I meant when you open and close the hamburger menu, not when the screen size changes.

Thanks for the help!

**Reply by abdullahtariq1171** [reply]
Because it changes `opacity` to zero when menu opens/closes. ([see `opacity-0` class](https://github.com/Edwardius/ed-eddy-edward/blob/main/components/nav/nav_hamburger/menu.tsx#L10))

Setting opacity to zero doesn't remove html from DOM.

**Reply by Edwardius** [reply]
so how would I fully remove the menu from the DOM after the transition to `opacity-0` ?
